src/components/backend-ai-scaling-group-list.ts

```diff
@@ -254,12 +297,23 @@ export default class BackendAIScalingGroupList extends BackendAIPage {
           id="controls"
           class="layout horizontal flex center"
         >
+        <wl-button fab flat inverted
+          icon="device_hub"
+          class="fg green"
+          @click=${() => {
+    this.selectedIndex = rowData.index;
+    this._launchDialogById('#keypair-detail-dialog');
+  }}
+      ><wl-icon>assignment</wl-icon></wl-button>
           <wl-button fab flat inverted
             icon="settings"
             class="fg blue"
             @click=${() => {
     this.selectedIndex = rowData.index;
     this.shadowRoot.querySelector('#modify-scaling-group-active').selected = this.scalingGroups[rowData.index].is_active;
+    Object.entries(JSON.parse(this.scalingGroups[this.selectedIndex].scheduler_opts)).map((item: any, index) => {
+      this._initializeSchedulerOpts(item[0], item[1]);
```

Comment:
because i change 'object in object' type to 'list in list' type to use map.
so, i approach key and value using list index. 
but, this approach is reduce readablity. so i change this logic